src/main/app/case/to-api-format.ts

```diff
@@ -32,6 +33,14 @@ const checkboxConverter = (value: string | undefined) => {
   return value === Checkbox.Checked ? YesOrNo.YES : YesOrNo.NO;
 };
 
+const checkboxConverterSwitchToSoleIntention = (value: string | undefined) => {
```

Comment:
I think we can just use the existing checkboxConverter

Reply:
Resolved

Reply:
fixed